packages/gatsby-codemods/package.json

```diff
@@ -24,16 +24,22 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@babel/runtime": "^7.12.5"
+    "@babel/runtime": "^7.12.5",
+    "jscodeshift": "^0.11.0"
   },
   "devDependencies": {
     "@babel/cli": "^7.12.1",
     "@babel/core": "^7.12.3",
     "babel-preset-gatsby-package": "^0.9.0-next.0",
-    "cross-env": "^7.0.2",
-    "jscodeshift": "^0.11.0"
+    "@babel/plugin-syntax-jsx": "^7.12.1",
+    "@babel/plugin-syntax-typescript": "^7.12.1",
+    "@babel/plugin-proposal-class-properties": "^7.12.1",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
```

Comment:
I'm late, but starting from Babel 7.8.0 this was enabled by default (because it reached Stage 4), so you don't need the syntax plugin anymore! ([blog post](https://babeljs.io/blog/2020/01/11/7.8.0#ecmascript-2020-default-support-10811httpsgithubcombabelbabelpull10811-10817httpsgithubcombabelbabelpull10817-10819httpsgithubcombabelbabelpull10819-10843httpsgithubcombabelbabelpull10843))